README.md

```diff
@@ -5,6 +5,9 @@
 
 Calibration software for HST/WFC3, HST/ACS, and HST/STIS.
 
+Nightly regression test results are available only from within the STScI network at this time.
```

Comment:
The endline in file does not really translate to endline in rendered markdown. FYI. I don't care personally but in case you do.